### ISSUE TITLE
weapons are not displayed when using manta style

### ISSUE THREAD

**Issue by alexasha991** [jeffhill/Dota2#417]
### Cosmetic Item Name

default weapon

### Description

when you use manta style, the illusions do not show weapons, I noticed this with huskar and enchantress

### Example Match ID

in game

### Screenshots

![image](https://user-images.githubusercontent.com/37921990/171884637-f87bf513-d7e3-4513-880d-4d69ecd015c8.png)
![image](https://user-images.githubusercontent.com/37921990/171885560-fac40769-3315-4551-805e-695ce71b926a.png)

**Comment by NMOTH** [comment]
cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.

**Comment by alexasha991** [comment]
> cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.

No, in demo mode everything works, it doesn't work on the server

**Comment by NMOTH** [comment]
> > cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.
> 
> No, in demo mode everything works, it doesn't work on the server

What do you mean by server? A private lobby created on valve servers? 
It still is just a visual glitch and not a reproducible bug.
can you verify integrity of your files and see if it happens still?

**Comment by alexasha991** [comment]
> > > cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.
> > 
> > 
> > No, in demo mode everything works, it doesn't work on the server
> 
> What do you mean by server? A private lobby created on valve servers? It still is just a visual glitch and not a reproducible bug. can you verify integrity of your files and see if it happens still?

checked the integrity of the files, and even launched the turbo game (I had to buy a manta style for Huskar), it's the same there

**Comment by NMOTH** [comment]
> > > > cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.
> > > 
> > > 
> > > No, in demo mode everything works, it doesn't work on the server
> > 
> > 
> > What do you mean by server? A private lobby created on valve servers? It still is just a visual glitch and not a reproducible bug. can you verify integrity of your files and see if it happens still?
> 
> checked the integrity of the files, and even launched the turbo game (I had to buy a manta style for Huskar), it's the same there

do you have a friend that can be on enemy team and help you test to check if this glitch is only visible to you or enemy as well?

**Comment by alexasha991** [comment]
> > > > > cannot reproduce. This is not a bug. Demo mode visual glitch sure. But not a game play bug.
> > > > 
> > > > 
> > > > No, in demo mode everything works, it doesn't work on the server
> > > 
> > > 
> > > What do you mean by server? A private lobby created on valve servers? It still is just a visual glitch and not a reproducible bug. can you verify integrity of your files and see if it happens still?
> > 
> > 
> > checked the integrity of the files, and even launched the turbo game (I had to buy a manta style for Huskar), it's the same there
> 
> do you have a friend that can be on enemy team and help you test to check if this glitch is only visible to you or enemy as well?

even the main character does not have a weapon
![image](https://user-images.githubusercontent.com/37921990/171938008-5d225a3f-6030-43c0-91ca-9a88fe158f01.png)
![image](https://user-images.githubusercontent.com/37921990/171938094-13e9deb8-2d14-4442-add1-6f6b696b6cef.png)

**Comment by NMOTH** [comment]
I can't reproduce this, we just have to wait to see if someone else has this issue. And under what conditions this is an issue.

**Comment by alexasha991** [comment]
> I can't reproduce this, we just have to wait to see if someone else has this issue. And under what conditions this is an issue.

i think this is due to body groups, the spear should disappear when it attacks, and particles along with it, i have a spear on enchanterss, and so there are no weapons on illusions, and when attacking, the particle does not disappear
![image](https://user-images.githubusercontent.com/37921990/171942050-51249f32-a4e5-4200-b161-73ee1a72eaf7.png)
